### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by lowjoel**
- Fix JavaScript lints
 - Fix bad HTML
 - Fix bad Slim syntax
 - Standardise DSL-style `fa_icon`

### CODE REVIEW

**Comment by lowjoel** [comment]
@allenwq might wanna review commit by commit, I think it should be quite harmless though.

**Comment by lowjoel** [comment]
And Hound is ignoring our JSHint config :angry:

**Comment by allenwq** [comment]
@lowjoel Looks good, will merge when it passed.

File: app/views/course/assessment/answer/programming/_test_cases.html.slim [hunk 1]
```diff
@@ -25,20 +25,20 @@ table.table.table-striped.table-hover
...
-          = fa_icon('check') if test_case_result && test_case_result.passed?
+          = fa_icon 'check'.freeze if test_case_result && test_case_result.passed?
   - if can?(:grade, answer.answer)
-    tbody
-      tr
-        th colspan=4
-          = t('.private')
-      - private_test_cases.each do |test_case|
-        = content_tag_for(:tr, test_case) do
+  tbody
```
Comment (lowjoel) [inline app/views/course/assessment/answer/programming/_test_cases.html.slim]: @allenwq wait, which one was correct?

**Pull request merged**